lib/src/traceable_widget.dart

```diff
@@ -21,6 +22,9 @@ class TraceableWidget extends StatefulWidget {
   /// {@macro traceableClientMixin.pvId}
   final String? pvId;
 
+  /// {@macro traceableClientMixin.updatePvIdAfterPop}
+  final bool updatePvIdAfterPop;
```

Comment:
As the mixin is already at `true` by default, it's better to have `updatePvIdAfterPop` nullable here so it can use the `super.updatePvIdAfterPop`